src/test/java/uk/gov/hmcts/juror/api/moj/report/standard/PersonAttendingSummaryReportTest.java

```diff
@@ -0,0 +1,148 @@
+import uk.gov.hmcts.juror.api.moj.domain.IJurorStatus;
+import uk.gov.hmcts.juror.api.moj.domain.QJurorPool;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReportTestSupport;
+import uk.gov.hmcts.juror.api.moj.report.DataType;
+import uk.gov.hmcts.juror.api.moj.repository.CourtLocationRepository;
+import uk.gov.hmcts.juror.api.moj.repository.PoolRequestRepository;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@SuppressWarnings("PMD.LawOfDemeter")
+class PersonAttendingSummaryReportTest extends AbstractStandardReportTestSupport<PersonAttendingSummaryReport> {
+
+    private CourtLocationRepository courtLocationRepository;
+
+    public PersonAttendingSummaryReportTest() {
+        super(QJurorPool.jurorPool,
+            PersonAttendingSummaryReport.RequestValidator.class,
+            DataType.JUROR_NUMBER,
+            DataType.FIRST_NAME,
+            DataType.LAST_NAME);
+    }
+
+    @BeforeEach
+    @Override
+    public void beforeEach() {
+        this.courtLocationRepository = mock(CourtLocationRepository.class);
+        super.beforeEach();
+    }
+
+    @Override
+    public PersonAttendingSummaryReport createReport(PoolRequestRepository poolRequestRepository) {
+        return new PersonAttendingSummaryReport(poolRequestRepository, this.courtLocationRepository);
+    }
+
+    @Override
+    protected StandardReportRequest getValidRequest() {
+        return StandardReportRequest.builder()
+            .reportType(report.getName())
+            .locCode(TestConstants.VALID_COURT_LOCATION)
+            .date(LocalDate.now())
+            .includeSummoned(false)
+            .build();
+    }
+
+    @Override
+    public void positivePreProcessQueryTypical(JPAQuery<Tuple> query, StandardReportRequest request) {
+        TestUtils.setupAuthentication("415", "COURT_USER", "1");
+
+        StandardReportRequest requestMock = mock(StandardReportRequest.class);
+        when(requestMock.getDate()).thenReturn(LocalDate.now());
+        when(requestMock.getLocCode()).thenReturn(TestConstants.VALID_COURT_LOCATION);
+
+        report.preProcessQuery(query, request);
+        verify(query, times(1))
+            .where(QJurorPool.jurorPool.nextDate.eq(requestMock.getDate()));
+        verify(query, times(1))
+            .where(QJurorPool.jurorPool.pool.courtLocation.locCode.eq(requestMock.getLocCode()));
+        verify(query, times(1))
+            .where(QJurorPool.jurorPool.status.status.in(IJurorStatus.RESPONDED,
+                                                         IJurorStatus.PANEL,
+                                                         IJurorStatus.JUROR));
+    }
```

Comment:
Needs more more test for when include Summoned is true

Reply:
Added